spec/examples/record/lazy_record_collection_spec.rb

```diff
@@ -8,7 +8,7 @@
       column :value, :text
     end
 
-    let(:now) { Time.now }
+    let(:now) { Time.now.round(3) }
     let(:event) { Event[now] }
 
     it { expect(event.attributes).to include(timestamp: now) }
```

Comment:
I dislike the explicit rounding. What about remove the round and doing something like:
```
... expect(event.timestamp).to be_with(1.millisecond).of(now) ...
```